xc/device.ml

```diff
@@ -565,7 +565,6 @@ module Vbd_Common = struct
       "mode", string_of_mode x.mode;
       "params", params;
       "qemu-params", qemu_params;
-      "direct-io-safe", "1";
```

Comment:
Stefano said this is what we want.

Reply:
Discussed that the way we use qemu would ignore this flag anyway, and we do not want to change the defaults that qemu uses otherwise when GFS2 is not in use.